content/en/account_management/rbac/log_management.md

```diff
@@ -13,61 +13,115 @@ further_reading:
+
+Because permissions are cumulative, if a user belongs to two roles, even if one of those roles grants a permission that is limited by a scope, if the other role grants the same permission without a scope limit, the user is granted that permission without being limited by any scope. 
+
+
+### Examples
+
+For example, if an organization has 4 roles with the following permissions and scopes:
+
+|  Role  |      Permission      |         Scope          |
+|--------|----------------------|------------------------|
+| Role 1 | logs_read_index_data | {}                     |
+| Role 2 | logs_read_index_data | {"indexes": "main"}    |
+| Role 3 | logs_read_index_data | {"indexes": "http"}    |
+| Role 4 | logs_read_index_data | {"pipelines": "12345"} |
+| Role 5 |                      |                        |
+
+Combinations of these roles would result in the following log read access:
+
+|    Roles     |              State                                                                                                                                  |
+|--------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| 1, any other | Has read access on all log indexes                                                                                                                  | 
+| 2, 5         | Has read access only on the "main" index                                                                                                            |
+| 2, 3, 5      | Has read access only on the "main" and "http" indexes                                                                                               |
+| 4, 5         | Has read access on no log indexes (the "logs_read_index_data" for Role 4 is limited to a pipeline, which means it grants no read access on indexes) |
 
-| Role                   | Default Access                                                                                                                          |
-|------------------------|-----------------------------------------------------------------------------------------------------------------------------------------|
-| Datadog Admin          | Can query on all log data and create or modify all log-related assets in a Datadog account.                                             |
-| Datadog Standard Role  | Can query on all log data and create or modify all log-related assets in a Datadog account. Can be modified to limit their permissions. |
-| Datadog Read Only Role | Can query on all log data in a Datadog account.  Can be modified to limit their permissions.                                            |
 
 ## Log Management Permissions
 
-The following permissions can be granted to manage read access on subsets of log data:
+### Reading Log Data
```

Comment:
my 2cts : let's do one section per permission instead. Because here you have to write some permissions twice. And side bonus, you can link the permission section to the main table for permissions in the Roles API page.